examples/worlds/graded_buoyancy.sdf

```diff
@@ -33,75 +33,13 @@
           <density>1</density>
         </density_change>
       </graded_buoyancy>
+      <white_list>ball_lighter_than_water</white_list>
```

Comment:
It's not clear if this refers to models or links. Can this be `enable_model` instead?

Reply:
Ah I meant for it to work with either the top level model or the link, as long as the scoped name is given. Nested models would be a bit more work. Let me improve this.

Reply:
Improved logic to support scoped names of models, links and nested models in 
fe6f894

Reply:
Ok, I'm also not a fan of using `whilte_list` and `black_list`. How about calling it `allow_list` or `enable_list` or something along those lines?

Reply:
Thats actually a good point... I think just `<enable>` could work as well.

Reply:
Ouch, you're right! Changed to `enable` in 6a83fac